ipfs-cluster-ctl/main.go

```diff
@@ -79,7 +79,7 @@ func main() {
 		cli.StringFlag{
 			Name:  "host, l",
 			Value: defaultHost,
-			Usage: "host:port of the IPFS Cluster service API",
+			Usage: "host:port of the IPFS Cluster service API\n\t\t",
```

Comment:
Hey Hector I am adding these line breaks because urfave appends the string describing default values to the usage string which causes the help description to overflow 80 characters, causing the sharness failure.  The suggested shortening won't prevent the 80 character overflow.